plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java

```diff
@@ -33,4 +35,46 @@ public void doGet(Request req, Response rsp, Context ctx) throws IOException {
                                 .toJson(r)
                                 .getBytes(StandardCharsets.UTF_8));
     }
+
+    /** Response for Model server endpoint */
+    public static class ExecutionParametersResponse {
+        @SerializedName("MaxConcurrentTransforms")
+        private int maxConcurrentTransforms;
+
+        @SerializedName("BatchStrategy")
+        private String batchStrategy;
+
+        @SerializedName("MaxPayloadInMB")
+        private int maxPayloadInMB;
+
+        public ExecutionParametersResponse() {
+            maxConcurrentTransforms = 4;
```

Comment:
NIT: Nothing to change immediately. I don't see a value for the constructor here. Seems like everything is being populated :).